### PR TITLE
update stripe package with running total model

### PR DESCRIPTION
**Pull request by fivetran-reneeli**
**Are you a current Fivetran customer?** 

Internal
**What change(s) does this PR introduce?** 

- Introduces a `running_total` model to alleviate runtime errors resulting from the daily_overview model exceeding disc capacity upon running.
**Did you update the CHANGELOG?** 

- [ ] Yes

**Does this PR introduce a breaking change?**

- [x] Yes (please provide breaking change details below.)
- [ ] No  (please provide an explanation as to how the change is non-breaking below.)

**Did you update the dbt_project.yml files with the version upgrade (please leverage standard semantic versioning)? (In both your main project and integration_tests)** 

- [x] Yes

**Is this PR in response to a previously created Bug or Feature Request**

- [x] Yes, Issue/Feature [[link bug/feature number here]](https://github.com/fivetran/dbt_stripe/issues/59)
- [ ] No 

**How did you test the PR changes?** 

 

- [x] BuildKite  
- [ ] Local (please provide additional testing details below)

**Select which warehouse(s) were used to test the PR**
 

- [x] BigQuery
- [x] Redshift
- [x] Snowflake
- [x] Postgres
- [x] Databricks
- [x] Other (provide details below)

**Provide an emoji that best describes your current mood**
 
:dancer:

**Feedback**

We are so excited you decided to contribute to the Fivetran community dbt package! We continue to work to improve the packages and would greatly appreciate your [feedback](https://www.surveymonkey.com/r/DQ7K7WW) on our existing dbt packages or what you'd like to see next.

### CODE REVIEW

File: models/stripe.yml [hunk 1]
```diff
@@ -1,6 +1,24 @@
...
+
+  - name: int_stripe__account_running_totals
+    description: Each record represents each day's ending balances per account, changes over time, and daily running (cumulative) balances. 
+
+  - name: int_stripe__date_spine
+    description: Each record represents a day of each calendar year.
+
+  - name: int_stripe__incomplete_charges
+    description: Each record represents a charge that is incomplete.
+
```
Comment (fivetran-catfritz) [inline models/stripe.yml:21]: This was helpful to understand what the different int models do!

**Review by fivetran-catfritz — COMMENTED** [review]
I was able to reproduce the resources exceeded bug by running the main branch, and running the new branch fixed it! So from that aspect I would say it's all working great!

Now that we've solved the issue, and thinking about what a new user might experience, I'm curious why this was moved to an int model? It seems like the final model is a straight select * from only the new int model, and since we're running int models as tables instead of ephemeral, we're generating two identical tables in the output (`int_stripe__account_running_total` is the same as `stripe__daily_overview`). 

If the reason is to keep the final model code clean, I think all we need to do is add a `{{ config(materialized='view') }}` to the `int_stripe__account_running_total` model. I think this could help since the other final models get the benefit of most of the int models being generated as tables, while here we avoid generating duplicate tables in the user's warehouse. I tested it out locally with a full refresh, and it panned out with our internal data, so let me know what you think!

Lastly just some small things. We need to update the installation version in the README, and I also left a few suggestions in in-line comments.

File: models/intermediate/int_stripe__account_rolling_totals.sql [hunk 2]
```diff
@@ -25,6 +25,7 @@ with date_spine as (
 ), final as (
 
     select
+        date_spine.account_id,
```
[the remark this answers is not in the document]
Comment (fivetran-reneeli) [reply models/intermediate/int_stripe__account_rolling_totals.sql:28]: true, added!

File: CHANGELOG.md [hunk 3]
```diff
@@ -1,3 +1,9 @@
+
+# dbt_stripe v0.10.0
+[#60](https://github.com/fivetran/dbt_stripe/pull/60) includes the following changes:
+## 🚨 Breaking Changes 🚨:
+- - Introduces a `int_stripe__account_running_totals` model to alleviate runtime errors resulting from the `stripe__daily_overview` model exceeding disc capacity upon running.
```
[the remark this answers is not in the document]
Comment (fivetran-reneeli) [reply CHANGELOG.md]: Good point, added more to the changelog!

**Comment by fivetran-reneeli** [comment]
Thanks for the review @fivetran-catfritz! I do agree with your point of there being 2 of the same tables basically. Therefore I added the view materialization config to the int model. Thanks for the suggestion!

File: models/stripe__daily_overview.sql [hunk 4]
```diff
@@ -1,32 +1,46 @@
...
-        date_day,
-        date_week,
-        date_month,
-        date_year,
-
-        {% for t in total_fields %}
-        {{ t }},
-        {% endfor %}
+        account_id,
+        {{ dbt_utils.generate_surrogate_key(['account_id','date_day']) }} as account_daily_id,
```
Comment (fivetran-catfritz) [inline models/stripe__daily_overview.sql:13]: Sorry I didn't catch this before, but I'm getting a test failure from 
```yml
    combination_of_columns:
            - date_day
            - source_relation
```
Guessing this test needs to be adjusted?

File: CHANGELOG.md [hunk 5]
```diff
@@ -1,3 +1,14 @@
+
+# dbt_stripe v0.10.0
+[#60](https://github.com/fivetran/dbt_stripe/pull/60) includes the following changes:
+## 🚨 Breaking Changes 🚨:
+- Introduces a `int_stripe__account_running_totals` model to alleviate runtime errors resulting from the `stripe__daily_overview` model exceeding disc capacity upon running.
+- Add `account_id` in `int_stripe__account_rolling_totals` for use as part of the join in the case where more than 1 `account_id` exists.
+
+## Under the Hood
+- The previous logic from `stripe__daily_overview` has been moved to the intermediate `int_stripe__account_running_totals` model to help optimize runtime.
```
Comment (fivetran-catfritz) [inline CHANGELOG.md:9]: I don't think some of this applies anymore.

Comment (fivetran-catfritz) [reply CHANGELOG.md:9]: And the README install sections still needs to be adjusted for the breaking change version.

File: models/stripe__daily_overview.sql (hunk 4 above)
Comment (fivetran-reneeli) [reply models/stripe__daily_overview.sql:13]: ah, added account_id, forgot I included that field now. Should be good now, thanks for catching!

**Review by fivetran-catfritz — APPROVED** [review]
Ran and tested locally. Reviewed updates. LGTM.

**Pull request merged**